tensorrt_llm/_torch/disaggregation/nixl/_agent_py.py

```
@@ -26,6 +26,14 @@ def is_completed(self):
         status = TransferState(self.agent.check_xfer_state(self.handle))
         return status == TransferState.DONE
 
+    def release(self):
+        try:
+            self.handle.release()
+            return True
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)